src/components/avatar/index.tsx

```diff
@@ -318,14 +362,61 @@ class Avatar extends PureComponent<AvatarProps> {
     }
   }
 
+  getText = memoize((label, name, getInitials) => {
+    let text = label;
+    if (_.isUndefined(label) && !_.isUndefined(name) && !_.isUndefined(getInitials)) {
+      text = getInitials(name);
+    }
+
+    return text;
+  });
+
+  get text() {
+    const {label, name, getInitials = AvatarHelper.getInitials} = this.props;
+    return this.getText(label, name, getInitials);
+  }
+
+  getBackgroundColor = memoize((text,
+    useHashBackgroundColor,
```

Comment:
Why go into this method if the user pass `useHashBackgroundColor: false`?

Reply:
Fixed